src/renderers/webvr/WebXRManager.js

```diff
@@ -113,6 +113,12 @@ function WebXRManager( renderer ) {
 
 	};
 
+	this.getSession = function () {
```

Comment:
Adding this back since it is used by `WebGLBackground` to get the session and determine if whether or not the background needs to be hidden based on the `xrSession.environmentBlendMode`

Reply:
Hmm, forgot about this...